presto-main/src/main/java/com/facebook/presto/execution/NodeResourceStatusConfig.java

```diff
@@ -35,4 +37,32 @@ public NodeResourceStatusConfig setRequiredWorkersActive(int requiredWorkersActi
         this.requiredWorkersActive = requiredWorkersActive;
         return this;
     }
+
+    @Min(0)
+    public int getRequiredResourceManagersActive()
+    {
+        return requiredResourceManagersActive;
+    }
+
+    @Config("cluster.required-resource-managers-active")
+    @ConfigDescription("Minimum number of active resource managers that must be available before activating the cluster")
+    public NodeResourceStatusConfig setRequiredResourceManagersActive(int requiredResourceManagersActive)
+    {
+        this.requiredResourceManagersActive = requiredResourceManagersActive;
+        return this;
+    }
+
+    @Min(0)
```

Comment:
Similar to Tim's above comment, may be this also could be @Min(1) ?

Reply:
updated